scripts/nayduck.py

```diff
@@ -50,6 +50,11 @@ def _parse_args():
                        '-i',
                        action='store_true',
                        help='Read test set from standard input.')
+    parser.add_argument(
+        '--dry-run',
+        '-n',
+        action='store_true',
+        help='Prints list of tests to execute, without doing anything')
```

Comment:
With https://github.com/near/nearcore/pull/5500 maybe this should be renamed to `-p --print`, `-o --output` or something similar and put in a new mutually exclusive group together with `--run-locally`?